src/cryptoadvance/specter/api/rest/api.py

```diff
@@ -0,0 +1,183 @@
+import logging
+import json
+
+import cryptoadvance.specter as specter
+from cryptoadvance.specter.api.rest.base import (
+    AdminResource,
+    BaseResource,
+    SecureResource,
+    rest_resource,
+)
+from cryptoadvance.specter.api.security import require_admin, verify_password
+from flask_restful import reqparse
+from flask import current_app as app
+from datetime import datetime
+from ...specter_error import SpecterError
+
+from .. import auth
+
+logger = logging.getLogger(__name__)
+
+parser = reqparse.RequestParser()
+parser.add_argument("greeting",
+                    help="This field cannot be blank",
+                    required=True)
+
+
+@rest_resource
+class ResourceHelloUser(SecureResource):
+    """ /api/v1alpha/hello """
+
+    endpoints = ["/v1alpha/hello"]
+
+    def get(self):
+        """ say hello to the User """
+        return {"hello": auth.current_user()}
+
+    def post(self):
+        """ greet the user """
+        args = parser.parse_args()
+        return {"greeting": args["greeting"]}
+
+
+@rest_resource
+class ResourceHelloAdmin(SecureResource):
```

Comment:
I guess this had a bug: It should derive from AdminResource, not SecureResource. I think that means that all the resources below should also derive from AdminResource!

Reply:
I think the "admin-is-the-only-guy-who-can-use-the-api" would work fine.